uisdk/src/main/res/values-en-rGB/strings.xml

```diff
@@ -12,6 +12,45 @@
+    <string name="kh_uisdk_adyen_payment_error_17">The specified PIN is incorrect or invalid.</string>
+    <string name="kh_uisdk_adyen_payment_error_18">The shopper specified an incorrect PIN more that three times in a row.</string>
+    <string name="kh_uisdk_adyen_payment_error_19">It is not possible to validate the specified PIN number.</string>
+    <string name="kh_uisdk_adyen_payment_error_20">The pre-authorisation risk checks resulted in a fraud score of 100 or more. Therefore, the transaction was flagged as fraudulent, and was refused.</string>
+    <string name="kh_uisdk_adyen_payment_error_21">The transaction was not submitted correctly for processing.</string>
+    <string name="kh_uisdk_adyen_payment_error_22">The sum of pre-authorisation and post-authorisation risk checks resulted in a fraud score of 100 or more. Therefore, the transaction was flagged as fraudulent, and was refused.</string>
+    <string name="kh_uisdk_adyen_payment_error_23">Transaction not permitted</string>
+    <string name="kh_uisdk_adyen_payment_error_24">The specified CVC (card security code) is invalid.</string>
+    <string name="kh_uisdk_adyen_payment_error_25">Restricted Card</string>
+    <string name="kh_uisdk_adyen_payment_error_26">Revocation of Authorization Order</string>
+    <string name="kh_uisdk_adyen_payment_error_27">Non Generic Error</string>
+    <string name="kh_uisdk_adyen_payment_error_28">The withdrawal amount permitted for the shopper\'s card has exceeded.</string>
+    <string name="kh_uisdk_adyen_payment_error_29">The number of withdrawals permitted for the shopper\'s card has exceeded.</string>
+    <string name="kh_uisdk_adyen_payment_error_31">Issuer reported the transaction as suspected fraud.</string>
+    <string name="kh_uisdk_adyen_payment_error_32">The address data the shopper entered is incorrect.</string>
+    <string name="kh_uisdk_adyen_payment_error_33">The shopper\'s bank requires the shopper to enter an online PIN.</string>
+    <string name="kh_uisdk_adyen_payment_error_34">The shopper\'s bank requires a checking account to complete the purchase.</string>
+    <string name="kh_uisdk_adyen_payment_error_35">The shopper\'s bank requires a savings account to complete the purchase.</string>
+    <string name="kh_uisdk_adyen_payment_error_36">The shopper\'s bank requires the shopper to enter a mobile PIN.</string>
+    <string name="kh_uisdk_adyen_payment_error_37">The shopper abandoned the transaction after they attempted a contactless payment and were prompted to try a different card entry method (PIN or swipe).</string>
+    <string name="kh_uisdk_adyen_payment_error_38">The issuer declined the authentication exemption request and requires authentication for the transaction. Retry with 3D Secure.</string>
+    <string name="kh_uisdk_adyen_payment_error_39">The issuer or the scheme wasn\'t able to communicate the outcome via RReq.</string>
```

Comment:
Not sure how these errors could be suitable for showing to the user
Also errors 37, 36, 26, 16, 18, 19, 03

Reply:
They were not very relevant for the user and may not be visible at any point, but they were in the documentation